cea/demand/comfort.py

```
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""
+Comfort models
+"""
+
+
+
+
+import numpy as np
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)